### DISCUSSION TITLE
Private Repo Forking

### DISCUSSION THREAD

**Discussion by sasikumarevature** [community/community#46478] [Repositories] [answered]
### Select Topic Area

Question

### Body

Can we fork private repo which does not belong to the organization?

**Comment by jge162** [comment, marked as answer]
well if the private repo is yours or you are invited to collaborate I do not see why not.

**Reply by sasikumarevature** [reply]
is there any possibility to fork private repo?

**Reply by jge162** [reply]
I think you need to be a collaborator. otherwise how can you fork it.

**Comment by sasikumarevature** [comment]
Hi, can we upload the repo as zip or tar files through git API?
